src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/Detail.html

```diff
@@ -51,6 +51,21 @@
             </div>
         </div>
 
+        <ul class="moving-column-alerts">
+            <li data-ng-repeat="(id, alert) in modals.alerts" class="moving-column-alert m-{{alert.mode}}" data-ng-click="modals.removeAlert(id)">
+                {{ alert.message | translate }}
+            </li>
+        </ul>
+
+        <div class="action-bar-modal" data-ng-if="modals.overlay">
```

Comment:
It is not a good idea to reuse `action-bar-modal` here. But we can do CSS refactoring later.